### DISCUSSION TITLE
Google字体API地址-修改后自动恢复默认-解决方法

### DISCUSSION THREAD

**Discussion by YukonKong** [mirai-mamori/Sakurairo#914] [Ideas]
![35E1C326629FDB387008103F9BDFFBF3](https://github.com/user-attachments/assets/6b931f72-c196-4db8-adc4-9591165bb211)
群中大佬解释：此选项的值硬编码在主题的PHP文件里（如上），需要手动替换

**Comment by letao9370** [comment]
亲测有效，首页的字体一下子就变了，F12里红色文字也消失，牛
<img width="52" alt="image" src="https://github.com/user-attachments/assets/28052f78-ee4e-44c8-8c4d-78e8d64c6952">
<img width="50" alt="image" src="https://github.com/user-attachments/assets/f0a3f735-6763-4a31-811c-9da3bd167b92">
